SalesforceMobileSDK.xcworkspace/xcshareddata/xcschemes/UnitTests.xcscheme

```diff
@@ -417,7 +417,7 @@
             </BuildableReference>
          </TestableReference>
          <TestableReference
-            skipped = "YES">
```

Comment:
enable unit tests for SalesforceHybridSDK (it was flappy but seems resolved after relax the timeout)